app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt

```diff
@@ -15,6 +15,11 @@ import kotlinx.coroutines.tasks.await
 import javax.inject.Inject
 
 private const val AQUARIUM_COLLECTION = "aquarium"
+private const val TAG = "AquariumRepository"
+
+const val WATER_LEVEL_CHANGE_DAILY = 0.1f
```

Comment:
I suggest changing this value to 0.25 so the result of not drinking water would be more clear to the user